drivers/test.go

```diff
@@ -295,7 +295,7 @@ func (td *TestDriver) Complete() {
 func (td *TestDriver) ApplyMessage(msg *types.Message) (result types.ApplyMessageResult) {
 	defer func() {
 		if r := recover(); r != nil {
-			result.Receipt.ExitCode = exitcode.SysErrInternal
+			result.Receipt.ExitCode = exitcode.SysErrForbidden
```

Comment:
Setting any exit code here at all is misleading. @frrist can we not?

If we must set one, maybe use SysErrReserved1 so that it can't possibly be confused with a legitimate code.

Reply:
Until we exercise test we expect to panic the `t.Fatal` ought to be enough